<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminPass</class>
 <widget class="QMainWindow" name="AdminPass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1290</width>
    <height>720</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0.327839, y1:0.644, x2:1, y2:0.004, stop:0.010929 rgba(58, 32, 142, 255), stop:0.538462 rgba(29, 59, 167, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="White" native="true">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>0</y>
      <width>540</width>
      <height>720</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.458158, y1:0.451, x2:0.448522, y2:0.78, stop:0.00549451 rgba(255, 255, 255, 255), stop:0.994505 rgba(255, 255, 255, 255));</string>
    </property>
    <widget class="QPushButton" name="LoginButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>490</y>
       <width>211</width>
       <height>71</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton {
	background-color: qlineargradient(spread:pad, x1:0.327839, y1:0.644, x2:1, y2:0.004, stop:0.010929 rgba(58, 32, 142, 255), stop:0.538462 rgba(29, 59, 167, 255));

    color: #ffdf00;
    font-family: 'Segoe UI', sans-serif;
    font-size: 15px;
    font-weight: bold;
    letter-spacing: 1px;
    text-transform: uppercase;
    padding: 13px 16px;
    border-radius: 8px;

    /* 3D outset-style border */
    border-style: solid;
    border-width: 5px;
    border-color: #ffffff #a0a0a0 #404040 #d0d0d0;  /* top, right, bottom, left */
}

QPushButton:hover {
    background-color: #1CB0F6;
	background-color: qlineargradient(spread:pad, x1:0.358436, y1:0.76, x2:0.999, y2:0.036, stop:0.010929 rgba(62, 55, 142, 255), stop:0.530055 rgba(74, 108, 167, 255));
}

QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:0.358436, y1:0.76, x2:0.999, y2:0.036, stop:0.010929 rgba(41, 32, 142, 255), stop:0.530055 rgba(37, 84, 167, 255));
    border-color: #404040 #d0d0d0 #ffffff #a0a0a0;
    padding-top: 16px;
    padding-bottom: 10px;
}

QPushButton:disabled {
    background-color: #b0b0b0;
    color: #f0f0f0;
    border-color: #cccccc;
}

</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BackButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>590</y>
       <width>211</width>
       <height>71</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {

	background-color: qlineargradient(spread:pad, x1:0.932234, y1:0.104, x2:0.1, y2:0.967803, stop:0.010929 rgba(176, 123, 123, 255), stop:0.530055 rgba(145, 15, 15, 255));
    color: #ffdf00;
    font-family: 'Segoe UI', sans-serif;
    font-size: 15px;
    font-weight: bold;
    letter-spacing: 1px;
    text-transform: uppercase;
    padding: 13px 16px;
    border-radius: 8px;

    /* 3D outset-style border */
    border-style: solid;
    border-width: 5px;
    border-color: #ffffff #a0a0a0 #404040 #d0d0d0;  /* top, right, bottom, left */
}

QPushButton:hover {
    background-color: #ff0000;
	background-color: qlineargradient(spread:pad, x1:0.932234, y1:0.104, x2:0.1, y2:0.967803, stop:0.010929 rgba(200, 123, 123, 255), stop:0.530055 rgba(120, 15, 15, 255));
}

QPushButton:pressed {
    /* Simulate button press by flipping light/shadow sides */
    border-color: #404040 #d0d0d0 #ffffff #a0a0a0;
    background-color: #780000;
    padding-top: 16px;
    padding-bottom: 10px;
}

QPushButton:disabled {
    background-color: #b0b0b0;
    color: #f0f0f0;
    border-color: #cccccc;
}
</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>171</width>
       <height>161</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;border:none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>coin.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="UsernameInput">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>360</y>
       <width>210</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    border: 1px solid #cccccc;
    border-radius: 6px;
    padding: 6px 10px;
    background-color: white;
    color: #333333;
    font-size: 14px;
    selection-background-color: #cce7ff;
}

QLineEdit:focus {
    border: 2px solid #0078d7;
    outline: none;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Admin ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="PassInput">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>410</y>
       <width>210</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    border: 1px solid #cccccc;
    border-radius: 6px;
    padding: 6px 10px;
    background-color: white;
    color: #333333;
    font-size: 14px;
    selection-background-color: #cce7ff;
}

QLineEdit:focus {
    border: 2px solid #0078d7;
    outline: none;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>160</y>
       <width>321</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent; border:none</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI','sans-serif'; font-size:36pt; font-weight:700; color:#ffdf00;&quot;&gt;Cryptex&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_6">
     <property name="geometry">
      <rect>
       <x>76</x>
       <y>159</y>
       <width>351</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent; border:none</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI','sans-serif'; font-size:38pt; font-weight:700; color:#000000;&quot;&gt;Cryptex&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_7">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>290</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; color:#000000;&quot;&gt;Enter your Admin ID and password&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <zorder>textEdit_6</zorder>
    <zorder>LoginButton</zorder>
    <zorder>BackButton</zorder>
    <zorder>label</zorder>
    <zorder>UsernameInput</zorder>
    <zorder>PassInput</zorder>
    <zorder>textEdit</zorder>
    <zorder>textEdit_7</zorder>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1290</width>
      <height>720</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;border:none</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>vecteezy_halftone-dotted-background_11887512.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label_3</zorder>
   <zorder>White</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
